File: WaterQuotaSystem/Models/ApplicationStatus.cs
namespace WaterQuotaSystem.Models;

public enum ApplicationStatus
{
    Accepted = 1,
    Processing = 2,
    Reviewing = 3,
    Approved = 4,
    Blocked = 5,
    ReturnedForEvidence = 6,
    Archived = 7,
    Timeout = 8
}

File: src/main/java/com/hospital/film/enums/ApplicationStatus.java
package com.hospital.film.enums;

public enum ApplicationStatus {
    ACCEPTED("已受理"),
    PROCESSING("处理中"),
    REVIEWING("复核中"),
    ARCHIVED("已归档"),
    REJECTED("已退回");

    private final String description;

    ApplicationStatus(String description) {
        this.description = description;
    }

    public String getDescription() {
        return description;
    }
}
